cli/src/utils/mod.rs

```diff
@@ -0,0 +1,75 @@
+/// converts string to bytes32
```

Comment:
don't have `utils` folder with undifferentiated code

Reply:
well this is a util, there is not certain place for it as far as I see it, can be used anywhere for converting a bytes32 to str and back,

Reply:
@rouzwelt so name it as such, make a module for converting bytes32